test/e2e/persistentips_test.go

```diff
@@ -261,20 +267,19 @@ var _ = Describe("Persistent IPs", func() {
 					WithPolling(time.Second).
 					ShouldNot(BeEmpty())
 
-				Expect(testenv.Client.Get(context.Background(), client.ObjectKeyFromObject(vmi), vmi)).To(Succeed())
-
-				Expect(vmi.Status.Interfaces).NotTo(BeEmpty())
-				Expect(vmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+				Expect(testenv.ThisVMI(vmi)()).Should(testenv.MatchIPs(ipsFrom, Not(BeEmpty())))
```

Comment:
wouldn't this be the same as 
```golang
Expect(...).NotTo(testenv.MatchIPs(ipsFrom, BeEmpty()))
```

Reply:
I think so, but both options seem equally convoluted. Let's tackle it in a follow up PR as you suggest